src/test/java/nl/tailormap/viewer/admin/ViewerAdminLockoutIntegrationTest.java

```diff
@@ -215,14 +246,14 @@ public void testCLockout() throws IOException, URISyntaxException {
         assumeFalse(null == is, "The catalina.out should privide a valid inputstream.");
 
         Scanner s = new Scanner(is);
-        boolean lokkedOut = false;
+        boolean lockedOut = false;
         while (s.hasNextLine()) {
             final String lineFromFile = s.nextLine();
-            if (lineFromFile.contains("An attempt was made to authenticate the locked user \"admin\"")) {
-                lokkedOut = true;
+            if (lineFromFile.contains(String.format("An attempt was made to authenticate the locked user \"%s\"", USER))) {
+                lockedOut = true;
                 break;
             }
         }
-        assertTrue(lokkedOut, "gebruiker 'admin' is buitengesloten");
+        assertTrue(lockedOut, String.format("gebruiker '%s' is buitengesloten", USER));
```

Comment:
you can actually use a closure here! so the string only gets lazily evaluated
```suggestion
        assertTrue(lockedOut, () ->"gebruiker '" + USER + "' is buitengesloten");
```

see https://junit.org/junit5/docs/current/user-guide/ (search for lazily)

Reply:
True, we programmers need to be lazy